js/core.js

```diff
@@ -57,21 +57,21 @@
      * URI should be to the ERMrest _service_. For example,
      * `https://www.example.org/ermrest`.
      */
-    function getServer(uri, params) {
+    function getServer(uri, contextHeaderParams) {
 
         if (uri === undefined || uri === null)
             throw new module.InvalidInputError("URI undefined or null");
 
-        if (typeof params === 'undefined' || params === null) {
+        if (typeof contextHeaderParams === 'undefined' || contextHeaderParams === null) {
```

Comment:
This `if` statement seems unnecessary, we're setting `cid` to null in the `Server` constructor.

Reply:
I am not sure why we're setting it as null. This was there from before. If we don't want to send the value then I can simply remove it.

Reply:
I think we're setting it to null so we can just take a look at the log and see if we have any cid=null or not, just to see if there are some cases that we forgot to pass cid. My point was that this logic is duplicated. [The constructor itself is doing the same](https://github.com/informatics-isi-edu/ermrestjs/pull/556/files#diff-d85c7ecdc144ac4aec3ee7a0b24c3f9fR104).

It's not an important change. We can leave it the way it is since it's a legacy code.